src/components/layout/LayoutSlots.tsx

```diff
@@ -0,0 +1,146 @@
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { PropsWithChildren } from 'react';
+import { cssVar } from '~utils/design-tokens';
+
+export enum GlobalGridArea {
+  header = 'header',
+  banner = 'banner',
+  sidebar = 'sidebar',
+  content = 'content',
+}
+
+export enum ContentGridArea {
+  header = 'header',
+  aside = 'aside',
+  page = 'page',
+}
+
+export enum PageGridArea {
+  header = 'header',
+  main = 'main',
+  footer = 'footer',
+}
+
+export enum AsideSize {
+  small = 'small',
+  medium = 'medium',
+  large = 'large',
+}
+
+const AsideWidth = {
+  [AsideSize.small]: cssVar('layout-aside-width-small'),
+  [AsideSize.medium]: cssVar('layout-aside-width-medium'),
+  [AsideSize.large]: cssVar('layout-aside-width-large'),
+};
+
+/*
+ * Layer 1 Grid components
+ */
+export const BannerContainer = styled.div`
+  grid-area: ${GlobalGridArea.banner};
+`;
+BannerContainer.displayName = 'BannerContainer';
+
+export const Header = styled.div`
+  grid-area: ${GlobalGridArea.header};
+`;
+Header.displayName = 'Header';
+
+export const ContentWrapper = styled.div<{ fixed?: boolean }>`
+  position: relative;
+  grid-area: ${GlobalGridArea.content};
+  overflow-y: hidden;
+
+  ${(props) =>
+    props.fixed
+      ? css`
+          margin-left: auto;
+          margin-right: auto;
+          max-width: 1550px;
+        `
+      : ''}
+
+  display: grid;
+  grid-template-columns: auto 1fr;
+  grid-template-rows: auto 1fr;
+  grid-template-areas:
+    '${ContentGridArea.header} ${ContentGridArea.header}'
+    '${ContentGridArea.aside} ${ContentGridArea.page}';
+`;
+ContentWrapper.displayName = 'ContentWrapper';
+
+/*
+ * Layer 2 Grid components
+ * (ContentGrid)
+ */
+interface AsideProps {
+  className?: string;
+  size: AsideSize;
+}
+export function Aside(props: PropsWithChildren<AsideProps>) {
+  const { children, size, ...restProps } = props;
+  return (
+    <StyledAside {...restProps} css={{ width: AsideWidth[size] }}>
+      {children}
+    </StyledAside>
+  );
+}
+const StyledAside = styled.div`
+  grid-area: ${ContentGridArea.aside};
+  background-color: rgba(10, 120, 30, 0.6);
+  padding: 16px;
+  width: 200px;
+  overflow-y: auto;
+`;
+StyledAside.displayName = 'StyledAside';
```

Comment:
I suggest to name it `AsideLeft`, in case we add an `AsideRight` in the future: we won't have to refactor as much